proxy_test/jobIDfile_gl

```diff
@@ -0,0 +1,53 @@
+###Submitted Job Ids###
```

Comment:
Did you accidentally add all files?  The job ID files should not be added (they're unique to each time you submit jobs).